cli/cook/format.py

```diff
@@ -36,13 +36,16 @@ def format_instance_status(instance):
     """Formats the instance status field"""
     status_text = format_state(instance['status'])
     if 'reason_string' in instance:
```

Comment:
It is possible for a task can have both `reason_string` and `progress` values? Do we want to avoid showing the progress in this scneario?

Reply:
Good call. I made it so that this field can accommodate either/both:

Both:
![image](https://user-images.githubusercontent.com/444778/32085164-1ab991ae-ba93-11e7-81dd-6dbd168304fd.png)

Just `reason_string`:
![image](https://user-images.githubusercontent.com/444778/32085226-59fc02e8-ba93-11e7-8e9a-ed82b8fdf131.png)

Just progress:
![image](https://user-images.githubusercontent.com/444778/32085234-6b00b8f4-ba93-11e7-910b-ed22eba6f60d.png)

Reply:
Done